### ISSUE TITLE
rstcheck.check has been removed

### ISSUE THREAD

**Issue by jondequinor** [equinor/subscript#484]
```python
    def test_hook_implementations_job_docs():
        """For each installed job, we require the associated
        description string to be nonempty, and valid RST markup"""
    
        plugin_m = ErtPluginManager(plugins=[subscript.hook_implementations.jobs])
    
        installable_jobs = plugin_m.get_installable_jobs()
    
        docs = plugin_m.get_documentation_for_jobs()
    
        assert set(docs.keys()) == set(installable_jobs.keys())
    
        for job_name in installable_jobs.keys():
            desc = docs[job_name]["description"]
            assert desc != ""
>           assert not list(rstcheck.check(desc))
E           AttributeError: module 'rstcheck' has no attribute 'check'
```

Seems 5.x.x is the last version of `rstcheck` that supports `check`.

**Comment by jondequinor** [comment]
I could not find a solution in that link @berland. Could you add the relevant part here?

**Comment by berland** [comment]
True, I did not find it myself :/

**Comment by berland** [comment]
https://rstcheck-core.readthedocs.io/en/latest/migrations/#rstcheck-v5-to-rstcheck-core-v1-rc1

Must probably rewrite to use functions from here: https://rstcheck-core.readthedocs.io/en/latest/autoapidoc/rstcheck_core/

**Comment by berland** [comment]
Solved in #492
